crates/sound/src/vhu_sound.rs

```diff
@@ -451,15 +469,29 @@ impl VhostUserSoundBackend {
 
         let audio_backend = alloc_audio_backend(config.audio_backend_name)?;
 
+        let stream_out_info = StreamInfo {
+            features : 0.into(),
+            formats : SUPPORTED_FORMATS.into(),
+            rates : SUPPORTED_RATES.into(),
+            direction : VIRTIO_SND_D_OUTPUT,
+            channels_min : 1,
+            channels_max : 6,
+        };
```

Comment:
Are the input/output streams have fixed channels/formats & everything? if that is the case then maybe doing something like 
```rust
impl StreamInfo {
    pub fn output() -> Self {
        Self {
            features : 0.into(),
            formats : SUPPORTED_FORMATS.into(),
            rates : SUPPORTED_RATES.into(),
            direction : VIRTIO_SND_D_OUTPUT,
            channels_min : 1,
            channels_max : 6,
        }
    }
}
```
would make things looks cleaner and self-documenting.

Reply:
Yes, it makes sense. I think in the future output() may require some parameters to set up rates and formats per stream but I think it is OK like this for the moment.